### ISSUE TITLE
[Feature]: Allow passing bounding box colors

### ISSUE THREAD

**Issue by vadimkantorov** [wandb/client#3318]
### Description

This is important to visually distinguish gt and pred detections without having every time to use the "eye" control.

Colors often can carry semantic meaning: 1) detection score, 2) IoU test success or failure etc

It also makes sense to allow to pass different colors for the box itself and for the box caption frame.

### Suggested Solution

N/A -> outlined abvoe

**Comment by ramit-wandb** [comment]
Hi @vadimkantorov,

Thanks for your feature request! I see an internal ticket similar to this already, I'll bump the priority on that for you.

Thanks,
Ramit

**Comment by vadimkantorov** [comment]
stale bump. I don't think it's very useful to close standing feature request as stale :(

**Comment by ramit-wandb** [comment]
Hi @vadimkantorov,

I'm sorry about this! This is one of our automations which closed the issue since there had been no activity here. The feature request is still planned, and this automation closing the ticket does not change that.

I'll look into the automation to try to prevent this behavior for feature requests we have.

Thanks,
Ramit

**Comment by MaxSchambach** [comment]
Indeed, this would be quite useful. It's a bit unfortunate, that it has been stale for over 1.5 years now 🥲
Any updates on this?

**Comment by ArtsiomWB** [comment]
Hey @MaxSchambach! Thank you for checking in! I checked internally and it looks like the feature request got backlogged because we were working on another potential solution at the time. If you would like to I would love to submit a feature request for you to bump this ticket and potentially raise the priority on it :)

**Comment by MaxSchambach** [comment]
@ArtsiomWB that would be great, thank you!

**Comment by ArtsiomWB** [comment]
Hey guys! I was notified that specifying colors for bounding boxes in the UI should be available sometime next week

**Comment by ArtsiomWB** [comment]
Hey @vadimkantorov! Looks like the changes just got pushed out!

<img width="431" alt="Image" src="https://github.com/user-attachments/assets/6d185ec2-c98c-486e-8834-ef6cbb397465" />
You should now be able to specify bounding box colors in the UI

**Comment by vadimkantorov** [comment]
In this issue I was proposing letting the user to specify the color of individual box from the code (along with caption), not in  the UI

E.g. correctly detected instances - green, incorrect detections - red

E.g. as in https://docs.pytorch.org/vision/main/generated/torchvision.utils.draw_bounding_boxes.html

**Comment by ArtsiomWB** [comment]
Hey @vadimkantorov, thank you for the follow up, that would be the next step for our engineering team, and I'll let you know as soon as I have any updates regarding this on my end.
